packages/manager/apps/pci-object-storage/src/pages/objects/container/object/show/useDatagridColumn.tsx

```diff
@@ -30,24 +38,46 @@ export const useDatagridColumn = ({
   const columns: DatagridColumn<TIndexedObject>[] = [
     {
       id: 'name',
-      cell: (props: TIndexedObject) => (
-        <div
-          className={props.isLatest ? 'is-latest' : props.versionId && 'ml-6'}
-        >
-          <div className="flex flex-col">
-            <DataGridTextCell>{props.name || props.key}</DataGridTextCell>
-            {props.isDeleteMarker && (
-              <OdsBadge
-                className="mt-3"
-                size="sm"
-                label={t(
-                  'pci_projects_project_storages_containers_container_delete_marker',
-                )}
-              />
-            )}
+      cell: (props: TIndexedObject) => {
+        const isLink = enableVersionsToggle
+          ? props.isLatest && shouldSeeVersions
+          : props.isLatest !== false &&
+            !isLocalZone &&
+            shouldSeeVersions &&
+            container.versioning?.status !== STATUS_DISABLED;
+
+        return (
+          <div
+            className={props.isLatest ? 'is-latest' : props.versionId && 'ml-6'}
```

Comment:
I don't understand this class

If isLatest the class will be is-latest
If not we check if versionId is truthy 
If not it is ml-6

It seems a bit complicated